source/calculus/source/08-SQ/07.ptx

```diff
@@ -103,19 +103,19 @@
             <p>
                 Find <m>\displaystyle\frac{a_{n+1}}{a_n}</m>.
                 <ol marker="A." cols="2">
-                    <li><m>\displaystyle \frac{n+1}{2}</m>.</li>                    
-                    <li><m>\displaystyle \frac{(n^2+1)(n+1)}{(n+2)n^2}</m>.</li>
-                    <li><m>\displaystyle \frac{(n+1)^2}{n+2}</m>.</li>
-                    <li><m>\displaystyle \frac{1}{2}</m>.</li>                    
-                    <li><m>\displaystyle \frac{(n+1)n^2}{n+2}</m>.</li>
+                    <li><m>\displaystyle 1+\frac{n+1}{n^2}</m>.</li>      <!-- bad subscript usage-->              
+                    <li><m>\displaystyle \frac{(n^2+1)(n+1)}{(n+2)n^2}</m>.</li><!-- freshman dream thm-->
+                    <li><m>\displaystyle \frac{(n+1)}{(n+2)n^2}</m>.</li><!-- naive cancel-->
+                    <li><m>\displaystyle \frac{(n+1)^3}{(n+2)n^2}</m>.</li>    <!-- ans -->                
+                    <li><m>\displaystyle \frac{(n+1)n^2}{n+2}</m>.</li> <!-- recip fail -->
                 </ol>                
             </p>
         </task>
         <task>
             <p>
                 Find <m>\displaystyle \lim_{n\to\infty} \left|\frac{a_{n+1}}{a_n}\right|.</m>
                 <ol marker="A." cols="2">
-                    <li><m>-\infty</m>.</li>
+                    <li><m>1</m>.</li><!-- ans -->
```

Comment:
Would be better to add an `<answer><p>A. <m>1</m></p></answer>` perhaps. Similarly, the comments above would be helpful as an `<answer/>` available in the instructor guide.

Reply:
Ok, I haven't done anything yet with adding answers for the instructor guide. I will look into that.

Reply:
basically you want `<task><statement><p>stuff</p></statement><answer><p>answer</p></answer></task>`

Reply:
@StevenClontz Thanks, I found it in the style guide too.